README.md

```diff
@@ -217,30 +265,31 @@ This means if the output directory has an active Terraform workspace, i.e. there
 
 ## How it Works
 
-`aztfexport` leverage [`aztft`](https://github.com/magodo/aztft) to identify the Terraform resource type on its Azure resource ID. Then it runs `terraform import` under the hood to import each resource. Afterwards, it runs [`tfadd`](https://github.com/magodo/tfadd) to generate the Terraform template for each imported resource.
+`aztfexport` leverages [`aztft`](https://github.com/magodo/aztft) to identify the Terraform resource type corresponding to an Azure resource ID. Then it runs `terraform import` under the hood to import each resource. Afterwards, it runs [`tfadd`](https://github.com/magodo/tfadd) to generate the Terraform HCL code for each imported resource.
 
 ## Demo
 
 [![asciicast](https://asciinema.org/a/sKYqzSiE5bpBJCB4BM2HjvF4j.svg)](https://asciinema.org/a/sKYqzSiE5bpBJCB4BM2HjvF4j)
 
-## Limitation
+## Limitations
 
-There are several limitations causing `aztfexport` can hardly generate reproducible Terraform configurations.
+We do not guarantee that redeploying Terraform configurations generated by `aztfexport` will reproduce the exact same infrastructure. This is due to known limitations that we detail below along with subsequently recommended actions. We will do our best to update this section as we discover new limitations.
 
-### AzureRM Provider Validation
+### Omitted Properties
 
 When generating the Terraform configuration, not all properties of the resource are exported for different reasons.
 
-One reason is because there are flexible cross-property constraints defined in the AzureRM Terraform provider. E.g. `property_a` conflits with `property_b`. This might due to the nature of the API, or might be due to some deprecation process of the provider (e.g. `property_a` is deprecated in favor of `property_b`, but kept for backwards compatibility). These constraints require some properties must be absent in the Terraform configuration, otherwise, the configuration is not a valid and will fail during `terraform validate`.
+One reason is because there are flexible cross-property constraints defined in the AzureRM Terraform provider. E.g. `property_a` conflicts with `property_b`. This might due to the nature of the API, or might be due to some deprecation process of the provider (e.g. `property_a` is deprecated in favor of `property_b`, but kept for backwards compatibility). These constraints require some properties to be absent in the Terraform configuration; otherwise, the configuration is not valid and will fail during `terraform validate`. For an example refer to [this schema for the azurerm_storage_blob resource](https://github.com/hashicorp/terraform-provider-azurerm/blob/8c7510d8b6ccf5098c7f05bd8c013a0e579a2e80/internal/services/storage/storage_blob_resource.go#L107-L126). If a user wishes to attempt to mitigate this issue, they would ultimately need to refer to the AzureRM documentation to know what, if any, properties are ommitted and what the current property value would entail.
```

Comment:
```suggestion
One reason is because there are flexible cross-property constraints defined in the AzureRM Terraform provider. E.g. `property_a` conflicts with `property_b`. This might due to the nature of the API, or might be due to some deprecation process of the provider (e.g. `property_a` is deprecated in favor of `property_b`, but kept for backwards compatibility). These constraints require some properties to be absent in the Terraform configuration; otherwise, the configuration is not valid and will fail during `terraform validate`.
```

The example is not appropriate here as we have some special handling to these constraints in the `tfadd`, where we don't want to expose the details here.